lexicon/providers/cloudflare.py

```diff
@@ -18,6 +18,7 @@ def ProviderParser(subparser):
 class Provider(BaseProvider):
 
     def __init__(self, options, engine_overrides=None):
+        print(options)
```

Comment:
extra print statement. should be removed.

Reply:
Right. Next commit.